cli/cmd/component-delete.go

```diff
@@ -127,10 +128,32 @@ func componentNamesToObjects(componentNames []string) ([]components.Component, e
 	return c, nil
 }
 
+func checkMutlipleReleaseNamespace(ns string, kubeconfig []byte, deleteNamespace bool) error {
```

Comment:
We should not add more code into `cli/cmd`. I'd make it part of `util.UninstallComponent`.